ui/store/actions.js

```diff
@@ -2819,46 +2817,6 @@ export function loadingTokenParamsFinished() {
   };
 }
 
-export function getTokenParams(address) {
```

Comment:
This method has been superceded by the more flexible/token standard agnostic method `getTokenStandardAndDetails` added on `metamask-controller.js`